cmd/boost/direct_deal.go

```diff
@@ -145,9 +179,9 @@ var directDealAllocate = &cli.Command{
 					Provider:   mid,
 					Data:       p.PieceCID,
 					Size:       p.Size,
-					TermMin:    verifregst.MinimumVerifiedAllocationTerm,
-					TermMax:    verifregst.MinimumVerifiedAllocationTerm,
-					Expiration: verifregst.MaximumVerifiedAllocationExpiration,
+					TermMin:    abi.ChainEpoch(cctx.Int64("termmin")),
+					TermMax:    abi.ChainEpoch(cctx.Int64("termmax")),
+					Expiration: abi.ChainEpoch(cctx.Int64("expiration")),
```

Comment:
Looks like these also need to be converted to snake-case.
Please test it out before merging.

Reply:
The basic command works. I am having trouble testing them E2E in devnet. I think Lotus side might be the issue as allocation is never created even when message is sent successfully.

Reply:
Ok I guess let's merge this PR.

For me it does seem to be working on a devnet:
```
boost$ boost allocate --miner=t01000 --pieceInfo=$COMMP_CID=8388608
about to send message with the following gas costs
max fee:      0.00000000166097256 FIL (absolute maximum amount you are willing to pay to get your transaction confirmed)
gas fee cap:  0.00000000000000012 FIL
gas limit:    13841438
gas premium:  0.000000000000000072 FIL
basefee:      0.0000000000000001 FIL

Proceed? Yes [y] / No [n]:
y
2023-09-05T10:17:07.049+0200	INFO	boost	boost/direct_deal.go:195	submitted data cap allocation message	{"cid": "bafy2bzacedevxmv2zlitcac2u4wbd6kiqlewkice7om24fhd3j52hdof2tcfa"}
2023-09-05T10:17:07.049+0200	INFO	boost	boost/direct_deal.go:196	waiting for message to be included in a block
AllocationID  Client  Miner  PieceCid                                                          PieceSize  TermMin  TermMax  
2             1007    1000   baga6ea4seaqamux3zrxsgljljnlnk45ztx3fxgggfrr3iffhqmniqfaf3ehscaa  8388608    518400   518400   
  Expiration: 172800

$ boost get-allocations
AllocationID  Client  Miner  PieceCid                                                          PieceSize  TermMin  TermMax  
2             1007    1000   baga6ea4seaqamux3zrxsgljljnlnk45ztx3fxgggfrr3iffhqmniqfaf3ehscaa  8388608    518400   518400   
  Expiration: 172800
```